### PR TITLE
Add compilation macro for oncn-mda

### PR DESCRIPTION
**Pull request by weli-l**
**What type of PR is this?**

**What this PR does / why we need it**:
Add compilation macro for oncn-mda
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: oncn-mda/cli_src/func/global.c [hunk 1]
```diff
@@ -158,11 +182,19 @@ static int get_map_fd(const char *const map_name, int *const fd)
     return SUCCESS;
 }
 
+#if LIBBPF_HIGHER_0_6_0_VERSION
+static int init_mesh_map(
+    struct mesh_map_info *const fds_map,
+    const char *const pin_file_path,
+    const char *const map_name,
+    struct create_map_attr *const map_attr)
+#else
```
Comment (supercharge-xsy) [inline oncn-mda/cli_src/func/global.c:191]: just include create_map_attr

Comment (weli-l) [reply oncn-mda/cli_src/func/global.c:191]: > just include create_map_attr

Yes, this can make code logic clearer，DONE

Comment (weli-l) [reply oncn-mda/cli_src/func/global.c:191]: > just include create_map_attr

Yes, this can make code logic clearer，DONE

**Pull request merged**